coalib/bearlib/languages/documentation/DocumentationComment.py

```diff
@@ -2,28 +2,32 @@
 
 
 @generate_repr()
-@generate_eq("documentation", "language", "docstyle", "marker", "range")
+@generate_eq("documentation", "language", "docstyle",
+             "indent", "marker", "range")
 class DocumentationComment:
     """
     The DocumentationComment holds information about a documentation comment
     inside source-code, like position etc.
     """
 
-    def __init__(self, documentation, language,
-                 docstyle, marker=None, range=None):
+    def __init__(self, documentation, language, docstyle,
+                 indent=None, marker=None, range=None):
```

Comment:
oh do you use actually as a value the whitespace-string or a number of spaces?
1. Describe that inside the docstring explicitly
2. If whitespace-string is used, you should have `""` as default value instead of `0`